### ISSUE TITLE
Batchnorm 

### ISSUE THREAD

**Issue by juhonkang** [juhonkang/machine_learning_articles#5]
## Short overview
Differentiate the batchnorm types:
Channel vs instance vs layer

### Article Link
[https://towardsdatascience.com/batch-normalization-in-3-levels-of-understanding-14c2da90a338#b93c](url)

## Key Takeaways
* Smoothen
* Faster training and better result
* The landscape to find optimal  get lesser local minimums

## Useful Code Snippets
```
python
#Add your codes here
import numpy as np
import pandas as pd
```

## Useful Tools
* 
*
